### DISCUSSION TITLE
Unknown scrape failure (Kemono)

### DISCUSSION THREAD

**Discussion by malik6524** [jbsparrow/CyberDropDownloader#305] [General]
I'm getting an outright scraping failure from kemono. Some pages work fine but most give this error and Cyberdrop ends up skipping all files. Any ideas? Been dealing with this since the importers for Kemono came back on. Thanks

**Comment by NTFSvolume** [comment]
Please attach your log file to debug the error. Also check if you are using the latest version and update if necessary. Latest is `v5.7.2`

**Reply by malik6524** [reply]
> Please attach your log file to debug the error. Also check if you are using the latest version and update if necessary. Latest is `v5.7.2`

Hi, thank you for replying. So I'm fairly new at this so I'm unsure which log files you want specifically so I'm attaching two here just in case.

[Download_Error_URLs.csv](https://github.com/user-attachments/files/17908274/Download_Error_URLs.csv)

[downloader.log](https://github.com/user-attachments/files/17908275/downloader.log)

And I am running the latest version as well.

**Reply by NTFSvolume** [reply]
Only 1 single URL failed with unknown. The rest were skipped because the were downloaded previously or cause the image was deleted from site (that's the 404)

If you try to download the image manually you will get the same error. 

I will need more time to find out why that particular URL failed with unknown. Maybe the title is different after they added the new importer.

**Reply by malik6524** [reply]
> Only 1 single URL failed with unknown. The rest were skipped because the were downloaded previously or cause the image was deleted from site (that's the 404)
> 
> If you try to download the image manually you will get the same error.
> 
> I will need more time to find out why that particular URL failed with unknown. Maybe the title is different after they added the new importer.

Thank you for the quick reply.

Also besides some links getting an unknown scrape failure some links also just stall at the beginning. You're right about these issues popping up after the new impoter. It seems like luck sometimes with links working and not.

Here's a screen I took with the hang.

![Screen Shot 2024-11-25 at 9 14 07 PM](https://github.com/user-attachments/assets/ef524814-d700-4ec5-b2b8-f98ba5b893ec)

>  ERROR    Scrape Failed: https://kemono.su/patreon/user/71084941 ('NoneType' object has no attribute 'text')                                                                                                     logger.py:21

**Comment by NTFSvolume** [comment]
Related to #320

**Comment by NTFSvolume** [comment]
Fixed in v5.7.2.post1
